LumpNav/LumpNav.py

```diff
@@ -211,8 +211,10 @@ def setupConnections(self):
     self.deleteAllFiducialsButton.connect('clicked()', self.onDeleteAllFiducialsClicked)
 
     self.rightCameraButton.connect('clicked()', self.onRightCameraButtonClicked)

```

Comment:
You should use lambda expressions:
self.rightCameraButton.connect('clicked()', lambda: self.cameraButtonClicked('View1') )

That way you would get rid of the intermediate onLeft/Right/CenterCameraButtonClicked methods.